src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,182 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.DisplayName.class)
+class RegistrationServiceImplTest {
+    private static final String STRING_123 = "123";
+    private static final String STRING_12345 = "12345";
+    private static final String STRING_12345678 = "12345678";
+    private static final String CORRECT_LOGIN = "123456";
+    private static final String CORRECT_PASSWORD = "654321";
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        user.setLogin(CORRECT_LOGIN);
+        user.setPassword(CORRECT_PASSWORD);
+        user.setAge(RegistrationServiceImpl.MIN_AGE);
+    }
+
+    @Test
+    @DisplayName("user equals null not Ok")
+    void register_nullUser_notOk() {
+        assertThrows(InvalidDataException.class, () -> registrationService.register(null));
+    }
+
+    @Test
+    @DisplayName("user already exist not Ok")
+    void register_alreadyExistUser_notOk() {
```

Comment:
Try to register **another** User with **same** login
```suggestion
    void register_alreadyExistLogin_notOk() {
```